apps/studio/components/interfaces/DiskManagement/DiskManagementForm.tsx

```diff
@@ -310,6 +311,35 @@ export function DiskManagementForm() {
           />
           {!isFlyArchitecture && (
             <>
+              <div className="flex flex-col gap-y-3">
+                <DiskCountdownRadial />
+                {usedPercentage >= 90 && isWithinCooldownWindow && (
```

Comment:
should it have `&& !isReadOnlyMode` too? Or is that checked in `isWithinCooldownWindow`

Reply:
ahhh yes great catch thank you so much! 🙏 will need to add this check or 2 warnings will show i think